src/walk-manifest.js

```diff
@@ -4,13 +4,18 @@ var url = require('url');
 var path = require('path');
 var fs = require('fs');
 
+// replace invalid http/fs characters with valid representations
+var fsSanitize = function(filepath) {
+	return filepath
+		.replace(/\?/g, '-questionmark-');
+};
```

Comment:
Pretty minor, but it might be nice to set a minimum node version for this project and use something 6+ so that we can do something like:

```
const fsSanitize = (filepath) => filepath.replace(/\?/g, '-questionmark-');
```

Reply:
Yeah I agree with you, but I think we should do it in another pr, it would probably help to add in the videojs-standard linter as well.